blog/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: blog
+  labels:
+    app: blog
+spec:
+  serviceName: blog
+  replicas: 1
+  selector:
+    matchLabels:
+      app: blog
+  template:
+    metadata:
+      labels:
+        app: blog
+    spec:
+{% if deploy %}
+      priorityClassName: production
+{% endif %}
+      containers:
+      - name: blog
+        image: ghost:3.0-alpine
+        env:
+          - name: NODE_ENV
+            value: production
+          - name: url
+{% if deploy %}
+            value: http://blog.hail.is
+{% else %}
+            value: http://internal.hail.is/{{ default_ns.name }}/blog
+{% endif %}
+        ports:
+        - containerPort: 2368
+        livenessProbe:
+          httpGet:
+{% if deploy %}
+            path: /
+{% else %}
+            path: /{{ default_ns.name }}/blog/
```

Comment:
If I set the ghost url to https://domain.com/path/to/subfolder/ is root / or https://domain.com/path/to/subfolder/?

Reply:
I believe Ghost's server listens on `/path/to/subfolder/`, if I understand your question/terminology correctly? Basically, if I set the ghost URL to `https://domain.com/path/to/subfolder/`, and I do (from the same machine that the ghost's server is on):
```
curl -v localhost:2368
curl -v localhost:2368/path/to/subfolder/
```
I get a 404 error for the first call, and the second call will return the homepage.

Reply:
Gotcha, thanks!